sql/analyzer/releaser.go

```diff
@@ -75,16 +75,16 @@ type releaseIter struct {
 func (i *releaseIter) Next() (sql.Row, error) {
 	row, err := i.child.Next()
 	if err != nil {
-		_ = i.Close()
+		_ = i.Close(nil)
```

Comment:
`Next(*sql.Context) (sql.Row, error)` wen?!

Reply:
I thought about it.  It's only going to get worse.  Should probably do it soon.